package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "ramble-fe",
+  "version": "0.0.1",
+  "description": "Ramble application front end",
+  "main": "index.js",
+  "scripts": {
```

Comment:
Great job using relitive paths for your npm scripts :)
however the npm test script should be pointing to karma :)